### ISSUE TITLE
{Linked:Bug9768646} [Adaptive Cards> Designer]: Screen reader is not announcing the label name for the fields.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8266]
### Target Platforms

Other

### SDK Version

1.5

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS Version: 22H2 (Build 25247.1000)
Browser: Edge Dev (111.0.1633.0)
Screen Reader: NVDA

**Repro Steps:**
1 Open the above URL in Edge Dev browser.
2. Navigate to the Card Payload Editor and press enter key to expand it.
3. Paste the attached JSON into it.
4. Navigate to Preview Mode button and press enter key.
5. Navigate to Purchase from edit field and observe the issue.

**Actual Result:**
While navigating to the 'Purchased From' edit field screen reader is not conveying the label information with it. It is announcing only value under the edit field.

**Expected Result:**
Screen Reader should convey the label information with the edit field, and it should announce as **'Purchased From required dollar 123 button'**..

**User Impact:**
Screen reader users may affect if it does not give accurate announcement to the action on the screen.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/Doc.aspx?sourcedoc=%7B3752B546-30D8-44D7-8221-6FC32EEB3AC8%7D&file=MAS%203.3.2%20%E2%80%93%20Labels%20or%20Instructions.docx&action=default&mobileredirect=true

"Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “.

### Screenshots

https://user-images.githubusercontent.com/95913748/215752603-2d023b29-65f9-41b8-a234-1c48669daa37.mp4

### Card JSON

```JSON
{
    "schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.6",
    "body": [
        {
            "separator": false,
            "spacing": "ExtraLarge",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Medium",
                    "color": "Default",
                    "weight": "Bolder",
                    "wrap": false,
                    "isSubtle": false,
                    "style": "heading",
                    "id": "",
                    "text": "Submit a claim"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "The information you are providing is collected here and sent to Forma for your Perks+ reimbursement."
                }
            ],
            "id": ""
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "emphasis",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "maxLines": 2,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Available balance: **Perks+ | $1,465.00**"
                }
            ],
            "id": ""
        },
        {
            "separator": false,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "TextBlock",
            "size": "Medium",
            "color": "Default",
            "weight": "Bolder",
            "wrap": false,
            "isSubtle": false,
            "style": "heading",
            "id": "",
            "text": "Enter claim details and save claim"
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "category",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Category"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Select category"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "CLICK_CHANGE_CATEGORY",
                    "selectedIndex": "${$index}"
                }
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "type",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Type"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Select type"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "CLICK_CHANGE_TYPE",
                    "selectedIndex": "${$index}"
                }
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "dateOfPurchase",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Date of purchase"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Select date of purchase"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "SET_DATE",
                    "selectedIndex": "${$index}"
                }
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "purchasedFrom",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Purchased from"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "$123"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "SET_FROM",
                    "selectedIndex": "${$index}"
                }
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "claimAmount",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Claim amount"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Enter claim amount"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "SET_AMOUNT",
                    "selectedIndex": "${$index}"
                }
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "description",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Description"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Enter description"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "SET_DESCRIPTION",
                    "selectedIndex": "${$index}"
                }
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "TextBlock",
            "size": "Medium",
            "color": "Default",
            "weight": "Bolder",
            "wrap": false,
            "isSubtle": false,
            "style": "heading",
            "id": "",
            "text": "Upload your receipts"
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "TextBlock",
            "size": "Small",
            "color": "Default",
            "weight": "Default",
            "wrap": true,
            "isSubtle": false,
            "style": "default",
            "id": "",
            "text": "Use as many as 5 files in these formats: .png, .jpg/jpeg, .tif/.tiff. Maximum file size is 5MB."
        },
        {
            "separator": false,
            "spacing": "ExtraLarge",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "ActionSet"
        },
        {
            "separator": false,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "emphasis",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "maxLines": 100,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "By clicking on the Submit button, I confirm that I have complete documentation and meet the requirements to receive reimbursement for Perks+. I confirm that the expenses for which I’m requesting reimbursement are for my own personal use (except where otherwise noted on HRWeb). I confirm all information is complete and accurate to the best of my knowledge. I understand if I return an item for which I have already received reimbursement through the Perks+ program, I must notify msft-perks@joinforma.com as soon as possible. I agree to work with Forma to return any money I received per the Perks+ guidelines for the returned item. I understand that failing to notify Forma about a returned item, is a violation of the Microsoft General Company Guidelines and may result in disciplinary action, up to and including termination of employment."
                }
            ],
            "id": ""
        },
        {
            "separator": false,
            "spacing": "ExtraLarge",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "ActionSet",
            "actions": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "left",
                    "verticalContentAlignment": "center",
                    "style": "positive",
                    "id": "SUBMIT_CLAIM_DETAILS",
                    "data": {
                        "id": "SUBMIT_CLAIM_DETAILS"
                    },
                    "type": "Action.Submit",
                    "title": "Submit"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "left",
                    "verticalContentAlignment": "center",
                    "id": "CANCEL_CLAIM_SUBMISSION",
                    "data": {
                        "id": "CANCEL_CLAIM_SUBMISSION"
                    },
                    "type": "Action.Submit",
                    "title": "Cancel"
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json"
}
```

**Comment by Raisul123** [comment]
Dependent Bug ID https://microsoftit.visualstudio.com/OneITVSO/_workitems/edit/9768646

**Comment by anna-dingler** [comment]
Hi @vagpt,

The expected behavior can be reached using `selectAction`'s `tooltip` and `title` properties.

Here is an updated JSON payload that behaves as expected with Narrator.

```JSON
{
    "schema": "http://adaptivecards.io/schemas/adaptive-card.json",
    "type": "AdaptiveCard",
    "version": "1.5",
    "body": [
        {
            "separator": false,
            "spacing": "ExtraLarge",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Medium",
                    "color": "Default",
                    "weight": "Bolder",
                    "wrap": false,
                    "isSubtle": false,
                    "style": "heading",
                    "id": "",
                    "text": "Submit a claim"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "The information you are providing is collected here and sent to Forma for your Perks+ reimbursement."
                }
            ],
            "id": ""
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "emphasis",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "maxLines": 2,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Available balance: **Perks+ | $1,465.00**"
                }
            ],
            "id": ""
        },
        {
            "separator": false,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "TextBlock",
            "size": "Medium",
            "color": "Default",
            "weight": "Bolder",
            "wrap": false,
            "isSubtle": false,
            "style": "heading",
            "id": "",
            "text": "Enter claim details and save claim"
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "category",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Category"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Select category"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "CLICK_CHANGE_CATEGORY",
                    "selectedIndex": "${$index}"
                },
                "title": "Category",
                "tooltip": "Select category"
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "type",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Type"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Select type"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "CLICK_CHANGE_TYPE",
                    "selectedIndex": "${$index}"
                },
                "title": "Type",
                "tooltip": "Select type"
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "dateOfPurchase",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Date of purchase"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Select date of purchase"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "SET_DATE",
                    "selectedIndex": "${$index}"
                },
                "title": "Date of purchase",
                "tooltip": "Select date of purchase"
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "purchasedFrom",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Purchased from"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "$123"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "SET_FROM",
                    "selectedIndex": "${$index}"
                },
                "title": "Purchased from",
                "tooltip": "$123"
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "claimAmount",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Claim amount"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Enter claim amount"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "SET_AMOUNT",
                    "selectedIndex": "${$index}"
                },
                "title": "Claim amount",
                "tooltip": "Enter claim amount"
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "RichTextBlock",
            "id": "description",
            "inlines": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": "Description"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextRun",
                    "size": "Small",
                    "color": "Attention",
                    "weight": "Default",
                    "wrap": false,
                    "isSubtle": false,
                    "text": " *"
                }
            ]
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "default",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Default",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "Enter description"
                }
            ],
            "id": "",
            "selectAction": {
                "type": "Action.Submit",
                "data": {
                    "id": "SET_DESCRIPTION",
                    "selectedIndex": "${$index}"
                },
                "title": "Description",
                "tooltip": "Enter description"
            }
        },
        {
            "separator": true,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "TextBlock",
            "size": "Medium",
            "color": "Default",
            "weight": "Bolder",
            "wrap": false,
            "isSubtle": false,
            "style": "heading",
            "id": "",
            "text": "Upload your receipts"
        },
        {
            "separator": false,
            "spacing": "default",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "TextBlock",
            "size": "Small",
            "color": "Default",
            "weight": "Default",
            "wrap": true,
            "isSubtle": false,
            "style": "default",
            "id": "",
            "text": "Use as many as 5 files in these formats: .png, .jpg/jpeg, .tif/.tiff. Maximum file size is 5MB."
        },
        {
            "separator": false,
            "spacing": "ExtraLarge",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "ActionSet"
        },
        {
            "separator": false,
            "spacing": "Large",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "Center",
            "type": "Container",
            "style": "emphasis",
            "items": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "Left",
                    "verticalContentAlignment": "center",
                    "type": "TextBlock",
                    "size": "Small",
                    "color": "Default",
                    "weight": "Default",
                    "wrap": true,
                    "maxLines": 100,
                    "isSubtle": false,
                    "style": "default",
                    "id": "",
                    "text": "By clicking on the Submit button, I confirm that I have complete documentation and meet the requirements to receive reimbursement for Perks+. I confirm that the expenses for which I’m requesting reimbursement are for my own personal use (except where otherwise noted on HRWeb). I confirm all information is complete and accurate to the best of my knowledge. I understand if I return an item for which I have already received reimbursement through the Perks+ program, I must notify msft-perks@joinforma.com as soon as possible. I agree to work with Forma to return any money I received per the Perks+ guidelines for the returned item. I understand that failing to notify Forma about a returned item, is a violation of the Microsoft General Company Guidelines and may result in disciplinary action, up to and including termination of employment."
                }
            ],
            "id": ""
        },
        {
            "separator": false,
            "spacing": "ExtraLarge",
            "horizontalAlignment": "Left",
            "verticalContentAlignment": "center",
            "type": "ActionSet",
            "actions": [
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "left",
                    "verticalContentAlignment": "center",
                    "style": "positive",
                    "id": "SUBMIT_CLAIM_DETAILS",
                    "data": {
                        "id": "SUBMIT_CLAIM_DETAILS"
                    },
                    "type": "Action.Submit",
                    "title": "Submit"
                },
                {
                    "separator": false,
                    "spacing": "default",
                    "horizontalAlignment": "left",
                    "verticalContentAlignment": "center",
                    "id": "CANCEL_CLAIM_SUBMISSION",
                    "data": {
                        "id": "CANCEL_CLAIM_SUBMISSION"
                    },
                    "type": "Action.Submit",
                    "title": "Cancel"
                }
            ]
        }
    ],
    "$schema": "http://adaptivecards.io/schemas/adaptive-card.json"
}
```

Let me know if you have any questions - thanks!

**Comment by vagpt** [comment]
Adding A11yTTValidated tag as issue is not repro.
